src/transaction/log_applier.c

```diff
@@ -8101,22 +8134,8 @@ la_apply_log_file (const char *database_name, const char *log_path, const int ma
   (void) os_set_signal_handler (SIGPIPE, SIG_IGN);
 #endif /* ! WINDOWS */
 
-  strncpy (la_slave_db_name, database_name, DB_MAX_IDENTIFIER_LENGTH);
-  s = strchr (la_slave_db_name, '@');
-  if (s)
-    {
-      *s = '\0';
-    }
-
-  s = la_get_hostname_from_log_path ((char *) log_path);
-  if (s)
-    {
-      strncpy (la_peer_host, s, CUB_MAXHOSTNAMELEN);
-    }
-  else
-    {
-      strncpy (la_peer_host, "unknown", CUB_MAXHOSTNAMELEN);
-    }
+  set_la_slave_db_name (la_slave_db_name, database_name);
+  set_la_peer_host (la_peer_host, log_path);
```

Comment:
la는 모듈 prefix이기 때문에 항상 제일 앞에 위치하는게 규약입니다.

```suggestion
  la_set_slave_db_name (la_slave_db_name, database_name);
  la_set_peer_host (la_peer_host, log_path);
```

Reply:
@hornetmj 반영하였습니다.